### ISSUE TITLE
Import defined in in-repo addon does not cause dependency to be included

### ISSUE THREAD

**Issue by jeanduplessis** [ef4/ember-auto-import#23]
If you define the import of a dependency in an in-repo addon, but not the main app, then the dependency is not included in the vendor.js file.

Using version 0.2.2 of ember-auto-import

**Comment by ef4** [comment]
Thanks for reporting. I think this is by design. In-repo addons are still addons, they get separate dependencies from their consuming app. Trying putting ember-auto-import into the in-repo addon's package.json in `dependencies` (no need to separately NPM install, it will find the copy from the parent app, but it won't use it unless it's listed in package.json).

**Comment by jeanduplessis** [comment]
@ef4 I tried adding it to the in-repo addon's package.json, but its still not including it in the vendor.js file.

**Comment by ef4** [comment]
In devDependencies or dependencies? Only dependencies should work.

**Comment by jeanduplessis** [comment]
In dependencies:

```json
"dependencies": {
  "ember-cli-babel": "~6.14.1",
  "ember-auto-import": "~0.2.2",
  "ua-parser-js": "~0.7.18"
}
```

**Comment by ef4** [comment]
Ok, sounds like a real bug then, I will investigate. Does the same package
work if you import it directly from the main app?

**Comment by jeanduplessis** [comment]
Yes, adding an import for the dependency anywhere in the main app causes it to be included in the vendor.js file and everything works as expected.

**Comment by jamescdavis** [comment]
Is `ua-parser-js` the package you are attempting to auto-import?

**Comment by jamescdavis** [comment]
I am seeing the same thing with 0.2.6, but this appears to be fixed in master.

**Comment by ef4** [comment]
I confirmed that this is fixed in 1.0.0-alpha.0.

**Comment by rickieshi** [comment]
Sry to bring up an old thread, but has anyone encountered the same issue with 1.5.3?
